scripts/word_embeddings/evaluate_fasttext_bin.py

```diff
@@ -0,0 +1,177 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Loading and evaluation of pretrained fasttext word embeddings
+================================================================
+
+This example shows how to load the binary format containing word and subword
+information and perform intrinsic evaluation of word embeddings trained with
+the facebookresearch/fasttext implementation using a variety of datasets all
+part of the Gluon NLP Toolkit.
+
+"""
+
+import argparse
+import logging
+import os
+import struct
+
+import mxnet as mx
+import numpy as np
+
+import evaluation
+import gensim
+import gluonnlp as nlp
+import utils
+
+
+def get_args():
+    """Construct the argument parser."""
+    parser = argparse.ArgumentParser(
+        description='Word embedding evaluation with Gluon.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    # Embeddings arguments
+    group = parser.add_argument_group('Embedding arguments')
+    group.add_argument('path', type=str,
+                       help='Path to pretrained TokenEmbedding file.')
+    group.add_argument(
+        '--max-vocab-size', type=int, default=None,
+        help=('Only retain the X first tokens from the pretrained embedding. '
+              'The tokens are ordererd by decreasing frequency.'
+              'As the analogy task takes the whole vocabulary into account, '
+              'removing very infrequent words improves performance.'))
+    group.add_argument('--list-embedding-sources', action='store_true')
+
+    # Computation options
+    group = parser.add_argument_group('Computation arguments')
+    group.add_argument('--batch-size', type=int, default=32,
+                       help='Batch size to use on analogy task.'
+                       'Decrease batch size if evaluation crashes.')
+    group.add_argument('--gpu', type=int, nargs='+',
+                       help=('Number (index) of GPU to run on, e.g. 0. '
+                             'If not specified, uses CPU.'))
+    group.add_argument('--no-hybridize', action='store_true',
```

Comment:
why would someone want to disable hybridization? Can we add more motivation in the help msg?